src/main/java/stringcalculator/PositiveStringNumber.java

```diff
@@ -0,0 +1,23 @@
+package stringcalculator;
+
+public class PositiveStringNumber {
+
+    private final int number;
+
+    public PositiveStringNumber(String stringNumber) {
+
+        try {
+            this.number = Integer.parseInt(stringNumber);
+        } catch (NumberFormatException e) {
+            throw new RuntimeException("invalid integer value: " + stringNumber);
+        }
+
+        if (this.number < 0) {
```

Comment:
매직넘버 0을 상수로 만들어서 의미를 부여해보면 좋겠어요!

```suggestion
        if (this.number < MIN_NUMBER) {
```